index.html

```diff
@@ -271,19 +270,26 @@ <h3 class="connect">LET'S CONNECT</h3>
               </li>
               <hr>
             </ul>
-            
-            
-
-            
-            
-              
-
-                  
-                  
-                
-                  
+              <!-------------------------------------- Contact form------------------------------->
+            <div class="about-contact">
+              <div class="title-container">
+                <h2 class="contact">Contact Me</h2>
+                <p class="para">If you have an application you are interested in<br>developing, 
+                  a feature that you need built or a project<br>that needs coding. i'd love to help with it</p>
+              </div>
+              <div class="form-container">
+                <form action="https://formspree.io/f/{form_id}" method="post" class="labels">
```

Comment:
- [ ] The padding on the forms needs a little bit of adjustment. It does not give a nice UI to have text starting on the margin attached.

## View 👍 
![image](https://user-images.githubusercontent.com/50990925/230001782-40166132-3da1-4a83-9e1f-b3fa6a333619.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/230002043-51ecc53c-763a-4e34-8452-3979d1f0f190.png)